lib/page/questions/questions_page.dart

```diff
@@ -57,7 +57,6 @@ class _QuestionsPageState extends ConsumerState<QuestionsPage> {
         return true;
       },
       child: Scaffold(
-        resizeToAvoidBottomInset: false,
```

Comment:
Why do we remove this?

Reply:
@luongvo We have to remove this to avoid the soft keyboard display on top of the below text fields 🙏🏼